<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	padding: 0px;
	margin: 0px;
	border: 0px;
}
QPushButton {
	font: 900 17pt &quot;Satoshi Black&quot;;
	background-color: transparent;
 	border-radius: 10px;
	font-weight: bold;
	color: #F4F7ED;
	text-align: left;
	padding: 5px 5px;
}
#NavBar {
    background-color: #2E6E65; 
}
QLabel {
	font: 900 10pt &quot;Satoshi Black&quot;;
	font-size: 20px;
	font-weight: bold;
	color: #2E6E65;
	qproperty-alignment: 'AlignLeft';
    qproperty-wordWrap: true;
	background: transparent;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="NavBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="DashBoardButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Images/resources.qrc">
          <normaloff>:/icons/icons/align-justify.svg</normaloff>:/icons/icons/align-justify.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="MainButtons">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="DashboardButton">
          <property name="text">
           <string> Dashboard</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/layout-dashboard.svg</normaloff>:/icons/icons/layout-dashboard.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CheckUpButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 900 20pt &quot;Satoshi Black&quot;;	
}
</string>
          </property>
          <property name="text">
           <string> Check Up</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/square-pen.svg</normaloff>:/icons/icons/square-pen.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RecordsButton">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string> Records</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/file-text.svg</normaloff>:/icons/icons/file-text.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string> Logout</string>
          </property>
          <property name="icon">
           <iconset resource="../Images/resources.qrc">
            <normaloff>:/icons/icons/log-out.svg</normaloff>:/icons/icons/log-out.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="MainBodyContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#Header {
	padding: 10px 20px;
padding-top: 15px;
}
QLabel {
    padding: 0px; 
}
#UserType {
	font: 300 10pt &quot;Lexend Light&quot;;
	font-size: 15px;
	color: #2B3752;
}
#UserName {
	font: 600 10pt &quot;Lexend SemiBold&quot;;
	font-size: 20px;
	color: #2B3752;
}
#InterfaceTitle{
	font-size: 35px;
}
QGraphicsView {
    border: 2px solid #ccc; 
    border-radius: 25px;
    background: white; 
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="Header">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="InterfaceTitle">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Check Up List</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="UserPanel">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="UserInfo">
             <property name="spacing">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="UserName">
               <property name="text">
                <string>Roy Adrian Rondina</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="UserType">
               <property name="text">
                <string>Doctor</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="Profile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="ProfilePic">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="MainBody">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#SearchBar {
	border: 2px solid #2E6E65;
	border-radius: 15px;
	text-align: center;
	background-color: transparent;
}

QComboBox {
    background-color: #b5cecb;
    font-size: 15px;
    color: #1b3b3a;
    border-radius: 10px;
    padding: 5px 10px;
    border: 1px solid #2E6E65;
}

QComboBox::drop-down {
    background: transparent;
    border: none;
    width: 20px;
}

#SeeAllButton {
	text-align: right;
	color: #2E6E65;
	font-size: 18px
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QFrame" name="SearchBarContainer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>70</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>11</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <property name="bottomMargin">
             <number>11</number>
            </property>
            <item>
             <widget class="QWidget" name="SearchBar" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">#SearchBar {
	background-color: #F4F7ED;
	border: 2px solid #2E6E65;
    border-radius: 15px;
}
#Search{
	background-color: transparent ;
	font: 300 14pt &quot;Lexend Light&quot;;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QPushButton" name="SearchButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../Images/resources.qrc">
                   <normaloff>:/icons/icons/search.svg</normaloff>:/icons/icons/search.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>35</width>
                   <height>35</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Search"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="SortByBox">
              <property name="styleSheet">
               <string notr="true">QComboBox {
    background-color: #D9E4DC;
    border: 2px solid #2E6E65;
    border-radius: 10px;
    padding: 5px 10px;
	font: 300 12pt &quot;Lexend Light&quot;;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left: 1px solid #2E6E65;
    background-color: #D9E4DC;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}
QComboBox::down-arrow {
    image: url(:/icons/icons/chevron-down.svg);
    width: 20px;
    height: 20px;
}
QComboBox QAbstractItemView {
    background-color: #F4F7ED;
    selection-background-color: #CCE3D0;
    border: 1px solid #2E6E65;
    border-radius: 2px;
    outline: 0;
	font: 300 12pt &quot;Lexend Light&quot;;
}</string>
              </property>
              <item>
               <property name="text">
                <string>Sort By:</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="SortOrderBox">
              <property name="styleSheet">
               <string notr="true">QComboBox {
    background-color: #D9E4DC;
    border: 2px solid #2E6E65;
    border-radius: 10px;
    padding: 5px 10px;
	font: 300 12pt &quot;Lexend Light&quot;;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left: 1px solid #2E6E65;
    background-color: #D9E4DC;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}
QComboBox::down-arrow {
    image: url(:/icons/icons/chevron-down.svg);
    width: 20px;
    height: 20px;
}
QComboBox QAbstractItemView {
    background-color: #F4F7ED;
    selection-background-color: #CCE3D0;
    border: 1px solid #2E6E65;
    border-radius: 2px;
    outline: 0;
	font: 300 12pt &quot;Lexend Light&quot;;
}</string>
              </property>
              <item>
               <property name="text">
                <string>Sort Order:</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="CheckUpLabel">
           <property name="text">
            <string>Check Ups</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	font: 900 10pt &quot;Satoshi Black&quot;;
	background-color:  #2E6E65;
	border-radius: 10px;
	font-weight: bold;
	font-size: 20px;
	color: #F4F7ED;
	text-align: center;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="AcceptedCheckUp">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget {
	background-color: #F4F7ED;
	gridline-color: transparent;
	border-radius: 10px;
	border: 2px solid #2E6E65;
}
QTableWidget::item {
	border: none;
	font: 16pt &quot;Lexend&quot;;
}
QTableWidget::item:selected {
	background-color: rgba(46, 110, 101, 0.3);
}
QTableWidget QHeaderView::section {
	background-color: #2E6E65;
	color: white;
	padding: 5px;
	font: 18px &quot;Lexend Medium&quot;;
	border: 2px solid #2E6E65;
}</string>
              </property>
              <column>
               <property name="text">
                <string>Check Up ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Status</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Check Up Type</string>
               </property>
              </column>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QFrame" name="frame_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>110</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="DiagnoseButton">
                 <property name="text">
                  <string>Diagnose</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="ModifyCheckUp">
                 <property name="text">
                  <string>Modify</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="SeeAllButton">
           <property name="styleSheet">
            <string notr="true">QPushButton {
text-allign: left;
}</string>
           </property>
           <property name="text">
            <string>Recent Completed Check Ups</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="DoneTable">
           <property name="styleSheet">
            <string notr="true">QTableWidget {
	background-color: #F4F7ED;
	gridline-color: transparent;
	border-radius: 10px;
	border: 2px solid #2E6E65;
}
QTableWidget::item {
	border: none;
	font: 16pt &quot;Lexend&quot;;
}
QTableWidget::item:selected {
	background-color: rgba(46, 110, 101, 0.3);
}
QTableWidget QHeaderView::section {
	background-color: #2E6E65;
	color: white;
	padding: 5px;
	font: 18px &quot;Lexend Medium&quot;;
	border: 2px solid #2E6E65;
}</string>
           </property>
           <column>
            <property name="text">
             <string>Check Up ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Diagnosis</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Check Up Date</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
